go/vm/lfvm/instructions_test.go

```diff
@@ -465,3 +466,116 @@ func TestMCopy(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateShanghaiInitCodeSize(t *testing.T) {
+	maxInitCodeSize := uint64(49152)
+	tests := []struct {
+		revision       vm.Revision
+		init_code_size uint64
+		expected       Status
+	}{
```

Comment:
If the change is introduced with Shanghai, I would have expected a test pre-shanghai, to make sure that the behavior has not been altered for older revisions.

Reply:
There are several test cases with Paris revision checking the difference between old revisions and Shanghai which introduced changes. Is this not sufficient?

Reply:
my bad, I confused the revision order again.